docs/versioned_docs/version-zenith/user/quickstart/532928-up.mdx

```diff
@@ -33,14 +33,14 @@ The `--native` flag results in the exposed ports on the container being auto-map
 However, you can modify this behavior and choose a different port on your host via the `--port` flag. For example:
 
 ```shell
-dagger up --port 9090:8080 service
+dagger call service up --ports 9090:8080
 ```
 
 This results in container port 8080 being mapped to host port 9090. You should now be able to access the application by browsing to `http://localhost:9090` (or equivalent host name) on the host.
 
 :::tip
 Here is an example of using this command with a module from the [Daggerverse](https://daggerverse.dev/):
 
-- `dagger up --native --mod github.com/helderco/daggerverse/python@kcna23 http-server --src <path/to/web/assets>`
+- `dagger call -m github.com/helderco/daggerverse/python@kcna23 http-server up --src <path/to/web/assets> --native`
```

Comment:
Fails, for @helderco 
```
┃ ImportError: cannot import name 'Annotated' from 'dagger.mod' 
```

Reply:
Remove `@kcna23`. That's fixed to last kubecon.

Reply:
Also, `up` is in the wrong place. `--src` is for `http-server` function:
```suggestion
- `dagger call -m github.com/helderco/daggerverse/python http-server --src <path/to/web/assets>  up --native`
```

Reply:
Fixed

Reply:
@vikram-dagger did you forget to merge the fixes here? The diff I'm seeing for the PR still shows `dagger call -m github.com/helderco/daggerverse/python@kcna23 http-server up --src <path/to/web/assets> --native`

Reply:
I see it.